docs/GettingStartedDocs/Contributors/InstallInfo.md

```diff
@@ -7,9 +7,14 @@ not currently need to be installed system-wide, so you could choose to install i
 into your home directory. From the build subfolder in your source tree:
 
 ```bash
-cmake -DCMAKE_INSTALL_PREFIX=~/openenclave-install ..
+cmake -G "Unix Makefiles" -DCMAKE_INSTALL_PREFIX=~/openenclave-install ..
```

Comment:
Do you think we need to specify the "Unix Makefiles"?  I think by default it uses make in my experience, but I don't really know. @andschwa what do you think?

Reply:
No, this is the default on Linux.

Reply:
@andschwa , adding "Unix Makefiles" is deliberate.  I'm trying to make a distinction between that and Ninja.  Besides, Unix Makefiles is not the default on windows (VC project is, I think) and that adds an additional degree of confusion.

Reply:
@andschwa Could I get an approval or a change request?

Reply:
Yes, on windows the default target is Visual Studio